entrypoint.sh

```diff
@@ -18,6 +18,7 @@ Environment Variables:
     SMF_DOMAIN - mail server hostname. use tutum/docker hostname if omitted.
     SMF_CONFIG - mail forward addresses mapping list.
     SMF_MYNETWORKS - configure relaying from trusted IPs, see http://www.postfix.org/postconf.5.html#mynetworks
+    SMF_SMARTHOST - configure a smarthost
```

Comment:
I suggest we should use `SMF_RELAYHOST` as the variable name as we are modifying the `relayhost` parameter of postfix.